map.html

```diff
@@ -75,6 +75,9 @@
 		<button id="button_Nirnroot" title="Shows where all Nirnroot are in the overworld.">Nirnroots</button>
 		<button id="button_ToggleTSP" title="The Traveling Salesman Path(TSP) is the fastest way to visit each location/nirnroot">Toggle Traveling Salesman Path</button>
 		<button onclick="showHelp()">Help!</button>
+		<span>Random gates found: 
+			<span id="random_Gate_Count">##</span> 
```

Comment:
make this camelCase. I know we've done a bad job of enforcing this.

Reply:
I was under the impression we do the HTML element Ids in this format "lower_Upper_Upper".

are we just removing the underscores? or do you mean something else?

Reply:
Yeah, remove the underscores. For the button_xxx's, it slipped by because the underscore kinda separated the type of the object from its name. So if this was count_randomGates, that might work. But that also is kinda weird because it implies that we're using count is a verb, not a noun. So randomGateCount is probably the best. (the button_xxx should probably be changed as well, but that's optional)